.gitignore

```diff
@@ -0,0 +1 @@
+/.idea
```

Comment:
I would prefer you would put this in ~/.gitignore as this is not generated by this project :D @codefromthecrypt

Reply:
the main risk of leaving out well known IDE paths is that pull requests from external devs end up including them. while I can do this, it is generally friendly to ignore widely used IDE files.

Reply:
done anyway, just that's why!

Reply:
hehe thanks, I understand your rationale! :D

Reply:
yeah because we will have to tell a potential noob this in various operating system dialects.

```bash
$ git config --global core.excludesFile '~/.gitignore'
$ echo .idea >> ~/.gitignore
```

but it maybe won't come up so we can wait and see!